aten/src/ATen/quantized/Copy.cpp

```diff
@@ -0,0 +1,20 @@
+#include <ATen/native/Copy.h>
+
+#include <ATen/ATen.h>
+#include <ATen/CPUApplyUtils.h>
+
+namespace at { namespace native {
+Tensor _s_copy_from_quantized(
```

Comment:
hm, I'd actually disallow explicit copy as it's unnatural and require calling int_repr (in other words - move this implementation to int_repr_quant